src/lib.rs

```diff
@@ -26,6 +26,10 @@
 //! which might be undesirable if the data are too large.
 //! An actual installer is still required if the binary involves too many resources that do not need to be kept in RAM all time.
 
+#![cfg_attr(not(any(test, feature = "std")), no_std)]
```

Comment:
Do we really need a feature gate for this? `#![no_std]` should be usable in binaries compiled with std as well.